.opensource/project.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "flank",
+    "platforms": ["Android", "iOS"],
+    "content": "README.md",
+    "pages" : ["faq.md", "contributing.md", "release_notes.md"],
+    "related": []
+  }
```

Comment:
missing new line

Reply:
hmmm above or below the bracket? @jan-gogo

Reply:
below, always add an empty line at the end of the file. You can configure IntelliJ to do it on save, just enable: 
``` Editor->General->Save files->Ensure an empty line at the end of a file on Save ```

Reply:
Below, it is visible if you open `Files changes` [tab](https://github.com/Flank/flank/pull/978/files)

Here you have explanation https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
> Below, it is visible if you open `Files changed` [tab](https://github.com/Flank/flank/pull/978/files)
> 
> Here you have explanation https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Interesting replies, I have had in the past debates about this but have not seen it from that perspective. I added it to intellij on auto save file as was commented (but now deleted) recommendation.

Reply:
Sorry for that delete I recommended enable in InteliJ 
```  Editor->General->Save File->Ensure an empty line at the end of a file on Save ```